libs/orders/src/lib/order-hooks/use-order-cancel.tsx

```diff
@@ -56,7 +69,7 @@ export const useOrderCancel = () => {
         });
 
         if (res?.signature) {
```

Comment:
chore (blocking): Either we need to gaurentee that order.id is present at this point or we need to determine the id of the order. If the first change the if, if the second change back the setId